src/context_store/retrieval/pipeline.py

```
@@ -0,0 +1,211 @@
+            len(graph_memories),
+        )
+
+        # ステップ 4: 結果統合 (RRF)
+        results_dict: dict[MemorySource, list[ScoredMemory]] = {
+            MemorySource.VECTOR: vector_results,
+            MemorySource.KEYWORD: keyword_results,
+            MemorySource.GRAPH: graph_memories,
+        }
+        fused = self.result_fusion.fuse_multiple_sources(results_dict, strategy)
+
+        # ステップ 5: fused_results を ScoredMemory に戻す（ID で lookup）
```

Ruff (0.15.7)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)